### PR TITLE
docs: add new demo example for highlight menu

### LINKED ISSUE #169: Document how-to highlight option
# General Support Request

See thread from listed issue

- https://github.com/AlaskaAirlines/auro-combobox/issues/57

## Support request

Update element documentation to illustrate a how-to feature for auro-menu that will allow a user to highlight an option in the menu.

## Possible Solution

There is already a feature that supports the pre-activation of an option in the menu, the scope of this update is to add more clarity to the documentation that illustrates how to use this in other use cases. 

## Exit criteria 

This issue will be considered complete once there is an updated demo that clarifies this existing solution.

### PR DESCRIPTION
**Pull request by irma-kurnia-alaska**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #169 

## Summary:

- Set the highlight menu option using `selected` attribute in auro-menuoptions

## Type of change:

- [x] Other (demo update) 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: apiExamples/selected.html [hunk 1]
```diff
@@ -1,10 +1,6 @@
-<auro-input id="matchWordInput" required>
-  <span slot="label">Enter a value to match in the menu</span>
-</auro-input>
-<br />
-<auro-menu id="matchWordMenu">
+<auro-menu>
```
Comment (jordanjones243) [inline apiExamples/selected.html:1]: Why did you remove the `matchWord` example? The highlight menu option should be an additional example. Removing this example has also caused your build to fail.

**Review by jordanjones243 — COMMENTED** [review]
You need to run your build to actually build out the documentation file to be used on the website. Right now, if you ran the localhost you don't even see the example.

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Accidentally submitted as a comment, but I am requesting changes.

File: apiExamples/selected.html (hunk 1 above)
Comment (irma-kurnia-alaska) [reply apiExamples/selected.html:1]: Hi @jordanjones243,

sorry I accidentally deleted that, i will back the example and add the example new for the highlight menu.

**Comment by irma-kurnia-alaska** [comment]
Hi @jordanjones243,

I will to rebuild docs now, thank you for your information.

**Comment by jordanjones243** [comment]
The work for the ticket attached to this will now be handled by this PR:
- https://github.com/AlaskaAirlines/auro-menu/pull/195
